README.md

```diff
@@ -14,6 +14,7 @@ A set of SVG maps that can be installed via [npm](https://www.npmjs.com) or [yar
 * 🇨🇻 [Cape Verde](packages/cape-verde)
 * 🇨🇳 [China](packages/china)
 * 🇨🇴 [Colombia](packages/colombia)
+* 🇩🇰 [Denmark](packages/denmark.regions)
```

Comment:
Do you plan to add another map of Denmark? (e.g. of [the municipalities](https://en.wikipedia.org/wiki/List_of_municipalities_of_Denmark))
If no, I think you can just call it `packages/denmark`

Reply:
I do not:) just foreseeing someone else wanting to create a more detailed map, thus leaving some space.
Will take your advice and rename it to "denmark". give me a few moments:)

Reply:
> just foreseeing someone else wanting to create a more detailed map, thus leaving some space

It also makes sense and I thought about it again after your comment. Since all the maps (except France, my fault 😅 ) use only the country name for the largest divisions (states, regions, counties...), I think it's better to keep it consistent.